ci/citest/acceptance-test/multibox/config.source

```diff
@@ -16,6 +16,11 @@ PREFIX="24"
 BOXES=(
     "minimal-7.2.1511-x86_64.kvm.box"
 )
+ZK_NODES=(
+  "10.0.100.10"
+  "10.0.100.11"
+  "10.0.100.12"
```

Comment:
It looks like all this array is used for is to output the list of zookeeper nodes when running the test. I think this is not a good way to do things. It needlessly clutters the configuration file.

The drawing in the illustrations folder should be enough to show where zookeeper is running.